example_dags/example_snowflake_partial_table_with_append.py

```diff
@@ -119,31 +109,24 @@ def example_snowflake_partial_table_with_append():
 
     filtered_data = filter_data(
         homes_long=transformed_data,
-        output_table=Table(name="expensive_homes_long"),
+        output_table=Table(),
+    )
+    homes_reporting = Table(conn_id=SNOWFLAKE_CONN_ID)
+    create_results_table = create_table(
+        table=homes_reporting, conn_id=SNOWFLAKE_CONN_ID
     )
-
-    create_results_table = create_table(conn_id=SNOWFLAKE_CONN_ID)
 
     # Append transformed & filtered data to reporting table
     # Dependency is inferred by passing the previous `filtered_data` task to `append_table` param
     # [START append_example_with_columns_list]
     record_results = append(
         source_table=filtered_data,
-        target_table=Table(name="homes_reporting", conn_id=SNOWFLAKE_CONN_ID),
+        target_table=homes_reporting,
         columns=["sell", "list", "variable", "value"],
     )
     # [END append_example_with_columns_list]
     record_results.set_upstream(create_results_table)
 
-    # We truncate this table only to avoid wasting Snowflake resources
-    # Why? Between 2022-03-25 and 2022-04-11 it accumulated 301G (89 million rows) because
-    # this example DAG used to append rows without deleting them
-    # [START drop_table_example]
-    truncate_results = drop_table(
-        table=Table(name="homes_reporting", conn_id=SNOWFLAKE_CONN_ID)
-    )
-    # [END drop_table_example]
-    truncate_results.set_upstream(record_results)
```

Comment:
Won't this break our truncate documentation?

Reply:
@tatiana  Nope it won't I added a separate example for this case in example_dags/example_sqlite_load_transform.py file.